### PR TITLE
Show Sac Value

### PR DESCRIPTION
**Pull request by LuxrayElite**
### Description:

solves issue #1334

Added visible sac value without sacrificing an item

### Changes:

Used a Variable of SacVal as a constant instead of making multiple calls (Fixed Value)
Returns full-length sac value and shorthand on no item provided eg  1,031,252,102 (1.03b)

### Other checks:

-   [ x] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/commands/Minion/sacrifice.ts [hunk 1]
```diff
@@ -38,12 +38,18 @@ export default class extends BotCommand {
 			excludeItems: msg.author.settings.get(UserSettings.FavoriteItems)
 		});
 
+		const SacVal = await msg.author.settings.get(UserSettings.SacrificedValue);
```
Comment (themrrobert) [inline src/commands/Minion/sacrifice.ts:41]: SacVal should be `sacVal` to match the style.

File: src/commands/Minion/sacrifice.ts [hunk 2]
```diff
@@ -38,12 +38,18 @@ export default class extends BotCommand {
...
+
 		if (!msg.author.owns(bankToSac)) {
 			return msg.channel.send(`You don't own ${bankToSac}.`);
 		}
 
 		if (bankToSac.length === 0) {
-			return msg.channel.send('No items found.');
+			return msg.channel.send(
+				`No items were provided. \nYour current sacrificed value is: ${SacVal.toLocaleString()} (${Util.toKMB(
```
Comment (themrrobert) [inline src/commands/Minion/sacrifice.ts:49]: No space needed before `\n`

**Review by themrrobert — CHANGES_REQUESTED** [review]

**Comment by LuxrayElite** [comment]
Suggested chaged by themrrobert have been made

File: src/commands/Minion/sacrifice.ts [hunk 3]
```diff
@@ -38,12 +38,18 @@ export default class extends BotCommand {
 			excludeItems: msg.author.settings.get(UserSettings.FavoriteItems)
 		});
 
+		const sacVal = await msg.author.settings.get(UserSettings.SacrificedValue);
```
Comment (gc) [inline src/commands/Minion/sacrifice.ts:41]: no `await`, `settings.get` doesnt return a promise.

**Review by gc — CHANGES_REQUESTED** [review]

**Pull request merged**